frontend/src/components/pages/search/components/FilterBar/ShowFilters.tsx

```diff
@@ -1,17 +1,56 @@
 import { SelectableDropdown } from 'components/pages/search/components/FilterBar/SelectableDropdown';
 import Field from 'components/pages/search/components/FilterBar/Field';
 import React from 'react';
-import { FilterState, Option } from '../../../../../modules/filters/interface';
+import { useIntl } from 'react-intl';
+import { DateFilter, FilterState, Option } from '../../../../../modules/filters/interface';
+import InputDateWithMagnifier from '../InputDateWithMagnifier';
 
 interface Props {
   item: FilterState;
   setFilterSelectedOptions: (filterId: string, options: Option[]) => void;
   hideLabel?: boolean;
+  dateFilter: DateFilter;
+  setDateFilter: any;
 }
 
-const ShowFilters: React.FC<Props> = ({ item, setFilterSelectedOptions, hideLabel = false }) => {
+const ShowFilters: React.FC<Props> = ({
+  item,
+  setFilterSelectedOptions,
+  hideLabel = false,
+  dateFilter,
+  setDateFilter,
+}) => {
+  const intl = useIntl();
+
+  if (item !== undefined && item.id === 'date-filter') {
+    return (
+      <div className="flex flex-col mt-4 desktop:mt-0 desktop:ml-5">
+        <div className="font-bold mb-2 text-lg">{intl.formatMessage({ id: 'search.diary' })}</div>
```

Comment:
In the return component, it is recommended to use `<FormattedMessage id="search.diary" />` but this is also correct 👍 .